src/transformers/image_utils.py

```diff
@@ -320,7 +320,7 @@ def load_image(image: Union[str, "PIL.Image.Image"], timeout: Optional[float] =
 
             # Try to load as base64
             try:
-                b64 = base64.b64decode(image, validate=True)
+                b64 = base64.decodebytes(image.encode() if isinstance(image, str) else image)
```

Comment:
So the fix is to make it handle string?

Reply:
But `image` is always `str` at this `if` branch `if isinstance(image, str):`, so I feel something is strange here.

Reply:
I still see we have 

> image.encode() if isinstance(image, str) else image

??